spec/controllers/sufia/admin/permission_templates_controller_spec.rb

```diff
@@ -40,5 +40,24 @@
         expect(flash[:notice]).to eq 'Permissions updated'
       end
     end
+    describe "update visibility" do
+      let(:admin_set) { create(:admin_set) }
+      let!(:permission_template) { Sufia::PermissionTemplate.create!(admin_set_id: admin_set.id) }
+      let(:input_params) do
+        { admin_set_id: admin_set.id,
+          sufia_permission_template: {
+            visibility: 'varies'
+          } }
+      end
+
+      it "is successful" do
+        expect(controller).to receive(:authorize!).with(:update, permission_template)
+        expect do
+          put :update, params: input_params
+        end.to change { permission_template.visibility }.from(nil).to('varies')
```

Comment:
NOTE: currently, this test is failing. Have the feeling I've missed something in my specs, as I'm not sure why it's failing.  The functionality "works" when I try it in the UI.